waspc/data/Generator/templates/sdk/wasp/auth/useAuth.ts

```diff
@@ -4,12 +4,13 @@ import { useQuery, addMetadataToQuery } from 'wasp/client/operations'
 import { api, handleApiError } from 'wasp/client/api'
 import { HttpMethod } from 'wasp/client'
 import type { AuthUser } from './types'
+import { UseQueryResult } from '@tanstack/react-query'
 
 // PUBLIC API
-export const getMe = createUserGetter()
+export const getMe: () => Promise<AuthUser | null> = createUserGetter()
 
 // PUBLIC API
-export default function useAuth(queryFnArgs?: unknown, config?: any) {
+export default function useAuth(queryFnArgs?: unknown, config?: any): UseQueryResult<AuthUser> {
```

Comment:
This function has a strange signature. I'll create an issue for investigating (out of scope for this PR).